governance.md

```diff
@@ -323,3 +323,19 @@ Existing **Tier 2** Partners:
 
 -   Bloomberg (Jason Grout, Sylvain Corlay)
 -   Google (Kester Tong, Kayur Patel, Mark Sandler)
+
+
+Changing the Governance
+=======================
+
+Changes to the governance documents are proposed via a GitHub pull
+request to The Project's governance documents GitHub repository at
+[https://github.com/jupyter/governance](https://github.com/jupyter/governance).
+After a period of public comment and review, a Steering Council Member
+proposes to the Steering Council that the pull request be merged
+(accepting the proposed changes) or proposes that the pull request be
+closed without merging (rejecting the proposed changes).  The Member
+should list the commit range proposed for acceptance or rejection and
```

Comment:
(To clarify my brief comment) - I think in this particular formal process, we should be very explicit about what is being voted on.  I also think it's fine for the steering council members to say that their vote applies to the proposed commit, as well as grammar fixes in the proposed commit, like in your recent proposal (so members can make their vote more broad than the proposed commit, or they can just vote for the proposed commit.)